pubspec.yaml

```
@@ -24,6 +24,9 @@ dependencies:
   flutter:
     sdk: flutter
   http: ^0.13.3
+  exported_movie_widgets:
+    git:
+      url: https://github.com/LeonardoMolleker/movie_widgets
 
```

```suggestion
  exported_movie_widgets:
    git:
      url: https://github.com/LeonardoMolleker/movie_widgets
      ref: main
```

you should include the reference to the branch inside of the repository, as the above example

---

Done